packages/suite/src/components/suite/WebUsbButton.tsx

```diff
@@ -1,20 +1,25 @@
 import TrezorConnect from '@trezor/connect';
 import { ButtonProps, Button } from '@trezor/components';
-import { Translation } from './Translation';
+import { Translation, TranslationKey } from './Translation';
 
-export const WebUsbButton = (props: Omit<ButtonProps, 'children'>) => (
+interface WebUsbButtonProps extends Omit<ButtonProps, 'children' | 'icon'> {
+    translationId?: TranslationKey;
+    icon?: ButtonProps['icon'] | 'NONE';
```

Comment:
I'd rather use 'false' instead of `'NONE'` because `'NONE'` could eventually be an icon name. Alternatively, you could add a `noIcon` prop.